csi/src/server.rs

```diff
@@ -109,8 +111,18 @@ impl AsyncWrite for UnixStream {
     }
 }
 
-#[tokio::main]
-async fn main() -> Result<(), String> {
+macro_rules! default_port {
+    () => {
+        10199
+    };
+}
+
```

Comment:
Not sure why we need a macro here. Why not just define a constant?

Reply:
Done. https://github.com/openebs/Mayastor/pull/462/commits/7c714899d911a2f659d5a7861968ba08b1aa354b